contrib/statesync.bash

```diff
@@ -1,33 +1,48 @@
 #!/bin/bash
 # microtick and bitcanna contributed significantly here.
+# Pebbledb state sync script.
+# invoke like: bash scripts/ss.bash
+
+## USAGE RUNDOWN
+# Not for use on live nodes
+# For use when testing.
+# Assumes that ~/.evmosd doesn't exist
+# can be modified to suit your purposes if ~/.evmosd does already exist
+
 set -uxe
 
 # Set Golang environment variables.
 export GOPATH=~/go
 export PATH=$PATH:~/go/bin
 
-# Install Gaia.
-make install
+# Install with pebbledb (uncomment for incredible performance)
+# go mod edit -replace github.com/tendermint/tm-db=github.com/baabeetaa/tm-db@pebble
+# go mod tidy
+go install -ldflags '-w -s -X github.com/cosmos/cosmos-sdk/types.DBBackend=pebbledb -X github.com/tendermint/tm-db.ForceSync=1' -tags pebbledb ./...
```

Comment:
If you want users to have the easiest experience, why install pebbledb and not use the default.
I would leave this commented out.
SDK are also phasing out some databases ...  do we want to include all of these, especially for new users who just want to try out Gaia and sync easily?

Reply:
sure